examples/java/build.gradle

```diff
@@ -161,3 +161,10 @@ task execute (type:JavaExec) {
   systemProperties System.getProperties()
   args System.getProperty("exec.args", "").split()
 }
+
+task wordCount(type:JavaExec) {
```

Comment:
Can you add a comment here, and the other wordcount tasks added, to explain that they're used for validating the development environment for contributors?